tests/functional/test_boilerplate.py

```diff
@@ -28,14 +29,21 @@ def test_error_raises_not_returns(backend):
 def test_quit_app(backend, firmware, navigator):
     if firmware.device.startswith("nano"):
         right_clicks = {'nanos': 3, 'nanox': 3, 'nanosp': 3}
+        backend.get_current_screen_content()
+
         for _ in range(right_clicks[firmware.device]):
             backend.right_click()
+            backend.wait_for_screen_change()
 
         with pytest.raises(ConnectionError):
             # clicking on "Quit", Speculos then stops and raises
             backend.both_click()
+            time.sleep(1)
```

Comment:
Why the sleep ?

Reply:
To wait for speculos to process the both_click action which ends-up in an app quit.
Previously this was done under the hood in the click. It's still the case but I think we shouldn't rely on it.